kuma/payments/utils.py

```diff
@@ -14,3 +14,10 @@ def popup_enabled(request):
     return (enabled(request) and
             hasattr(request, 'user') and
             flag_is_active(request, CONTRIBUTION_BETA_FLAG))
+
+
+def recurring_payment_enabled(request):
+    """Returns True if recurring payment is enabled for the user."""
+    return (enabled(request) and
+            hasattr(request, 'user') and
+            flag_is_active(request, RECURRING_PAYMENT_BETA_FLAG))
```

Comment:
Recurring payments should be disabled if the payments beta flag is disabled. Like this:

```
    return (enabled(request) and
            hasattr(request, 'user') and
            flag_is_active(request, CONTRIBUTION_BETA_FLAG) and
            flag_is_active(request, RECURRING_PAYMENT_BETA_FLAG))
```
or shorter:
```
    return (popup_enabled(request) and 
            flag_is_active(request, RECURRING_PAYMENT_BETA_FLAG))
```